events/wc_offmap_events.txt

```diff
@@ -958,4 +958,222 @@ character_event = {
 			}
 		}
 	}
+}
+#Healed by Old Gods
+character_event = {
+	id = WCOME.500
+	picture = GFX_evt_old_god_healing
+	desc = EVTDESC_WCOME.500
+	border = GFX_event_normal_frame_religion
+	
+	is_triggered_only = yes
+	
+	only_playable = yes
+	
+	#Heal wounds
+	option = {
+		name = EVTOPTA_WCOME.500
+		trigger = {
+			has_injury_trigger = yes
+		}
+		heal_wounds_magic_effect = yes
+		hidden_effect = {
+			random_list = {
+				15 = {
+					trigger = { can_get_mutation_healing_trigger = yes }
+					character_event = { id = WCVOD.300 }
+				}
+				85 = {}
+			}
+		}
+	}
```

Comment:
Disable this option if other options are available because either the player or AI can pick it though another option heals both wounds and diseases.